librz/debug/dsignal.c

```diff
@@ -85,43 +85,58 @@ static bool siglistcb(void *p, const char *k, const char *v) {
 	return true;
 }
 
+struct DebugPJ {
```

Comment:
I would use `debug_pj`, as in general we use those names unless it is for exported things.